core/src/main/java/org/apache/calcite/sql/SqlDialectFactoryImpl.java

```diff
@@ -221,6 +221,8 @@
       return VerticaSqlDialect.DEFAULT;
     case SPARK:
       return SparkSqlDialect.DEFAULT;
+    case SNOWFLAKE:
```

Comment:
Alphabetic order? Looks the SPARK out of position too?

Reply:
Good advice, I've changed it.